src/utils/camel.ts

```diff
@@ -1,13 +1,19 @@
-export function snakeToCamel(str: string): string {
+/**
+ * @private
+ **/
```

Comment:
oesn't matter yet, but added @Private while I was at it. This ensures it wont show up in TSDocs.